english/pick-up-blowtorch/blowtorch.md

```diff
@@ -21,4 +21,6 @@ Turning up your blowtorch, you make a mental note not to cross the streams and y
 into the room screaming.
 
 You're not sure if you've got enough graham cracker and chocolate for this, but at least you're 
-bringing the camp fire... 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.